macro.d.ts

```diff
@@ -0,0 +1,3 @@
+import { ICompilerOptions } from "."
+import { ICheckerSuite } from "ts-interface-checker"
+export declare function makeCheckers (modulePath?: string, options?: ICompilerOptions): ICheckerSuite
```

Comment:
You are still planning to add `buildTypeSuite` too, right?

Reply:
Yes, but I decided to go with `getTypeSuite` instead of `buildTypeSuite`, and I also renamed `makeCheckers` to `getCheckers`. You will know why when you see how I solved my first 2 TODO points in the macro output: https://github.com/gristlabs/ts-interface-builder/pull/24/files?file-filters%5B%5D=.json#diff-97d2154d09a1b7f451079cd3af659ebf

Reply:
`getTypeSuite` is now implemented so I'll resolve this